### ISSUE TITLE
[Bug?][Fabric 1.21.1] Sunken skeletons drown underwater

### ISSUE THREAD

**Issue by KorvasSol** [Mlodziak00/Skeletal-Remains-Fabric#12]
I don’t know if this is intentional or not but it seems unlikely that it is but Sunken skeleton variants seem to drown after being underwater for an extended period of time.

**Comment by Mlodziak00** [comment]
Yes that's a bug im trying to get to, wasn't able to work in stuff, all bugs will be fixed as soon as I can work again
